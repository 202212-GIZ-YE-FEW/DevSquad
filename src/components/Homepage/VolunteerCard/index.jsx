import AOS from "aos";
import Image from "next/image";
import { useEffect } from "react";

import "aos/dist/aos.css";
const VolunterCard = ({ volunteerImage, name, voldescribtion }) => {
    useEffect(() => {
        AOS.init();
    }, []);
    return (
        <div>
            <div className='flex flex-col items-center font-Rubik	'>
                <div className='relative w-20 h-20 '>
                    <Image
                        src={volunteerImage}
                        alt='profile'
                        fill
                        className=' object-contain rounded-full '
                    />
                </div>
                <div className='flex flex-col gap-2  pt-6'>
                    <p className='text-xl text-center font-medium'>{name}</p>
                    <p className='text-lg text-primary-gray text-center font-normal '>
                        {voldescribtion}
                    </p>
                </div>
            </div>
        </div>
    );
};

export default VolunterCard;
