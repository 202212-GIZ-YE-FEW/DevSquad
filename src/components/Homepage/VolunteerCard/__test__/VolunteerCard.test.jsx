import renderer from "react-test-renderer";
import VolunteerCard from "../Index";

it("renders correctly", () => {
    const tree = renderer.create(<VolunteerCard />).toJSON();
    expect(tree).toMatchSnapshot();
});
