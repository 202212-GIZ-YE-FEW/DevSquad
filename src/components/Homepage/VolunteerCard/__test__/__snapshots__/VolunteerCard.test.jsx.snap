// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="flex flex-col items-center font-Rubik "
  >
    <div
      className="relative w-20 h-20"
    >
      <img
        alt="profile"
        className="object-contain rounded-xl"
        data-nimg="fill"
        decoding="async"
        loading="lazy"
        onError={[Function]}
        onLoad={[Function]}
        src=""
        style={
          Object {
            "bottom": 0,
            "color": "transparent",
            "height": "100%",
            "left": 0,
            "objectFit": undefined,
            "objectPosition": undefined,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "width": "100%",
          }
        }
      />
    </div>
    <div
      className="flex flex-col gap-2  pt-6"
    >
      <p
        className="text-xl text-center font-medium"
      />
      <p
        className="text-lg text-primary-gray text-center font-normal "
      />
    </div>
  </div>
</div>
`;
